cypress/e2e/helpers/surveyWebPatientHelpers.ts

```diff
@@ -0,0 +1,84 @@
+import moment from "moment";
+import "moment/min/locales";
+
+import * as assignSurvey from "../helpers/adminHelpers/assignSurvey";
+
+export const DATE_TEXT: string = "p.MuiTypography-body1.css-xhbppr";
+export const CLOSE_BUTTON: string = "button";
+export const VALUE_LBS_INPUT: string = "input.MuiInputBase-input";
+export const LBS_TEXT: string = "p.MuiTypography-root.css-10o8u6h";
+export const PROGRESS_BAR_50_IMG: string =
+  'span[role="progressbar"][aria-valuenow="50"]';
+export const PROGRESS_BAR_100_IMG: string =
+  'span[role="progressbar"][aria-valuenow="100"]';
+
+export const VALUE_PROGRESS_BAR_TEXT: string = "p.MuiTypography-root";
+export const NEXT_BUTTON: string = "button.MuiButton-containedPrimary";
+
+export const SNACK_BAR_ALERT_ID: string = '[data-testid="SNACK_BAR_ALERT"]';
+
+export const verifySurveyCard = () => {
+  cy.contains("h6", "SER-ISD-001 - Body Weight");
+  cy.contains("h6", "OPEN");
+
+  const currentDate = moment().format("MMM D");
+  const expectedText = `Once on ${currentDate} at ${assignSurvey.calculatedTime}`;
```

Comment:
Here we could create a function to return the correct string

const SCHEDULE_ONCE_TEXT = (date: string, hour: string) => `Once on ${date} at ${hour}`
```suggestion
  cy.contains("h6", SCHEDULE_ONCE_TEXT(date, hour));
```